plugins/copr.py

```diff
@@ -520,13 +553,9 @@ def _disable_repo(self, copr_username, copr_projectname):
                 _("Failed to disable copr repo {}/{}"
                   .format(copr_username, copr_projectname)))
 
-        self.base.conf.write_raw_configfile(repo.repofile, repo.id,
-                                            self.base.conf.substitutions, {"enabled": "0"})
-
-        # disable also multilib counterpart, if found
-        multilib_repo_id = repo.id + ':ml'
-        if multilib_repo_id in self.base.repos:
-            self.base.conf.write_raw_configfile(repo.repofile, multilib_repo_id,
```

Comment:
The multilib part here was removed... I guess this is intentional but seems unrelated to the commit, it probably should go into a separate commit with its own justification for removal?

Reply:
(Looking at it some more it seems this is a hack that is being removed by a generic solution - worth putting into the commit message as well)